tests/stress_tests/common/CMakeLists.txt

```diff
@@ -27,11 +37,69 @@ if(EXISTS "${OpenVINO_SOURCE_DIR}/thirdparty/gflags")
     add_gflags()
 endif()
 
+#
+# Google Tests framework
+#
+if(EXISTS "${OpenVINO_SOURCE_DIR}/thirdparty/gtest")
```

Comment:
better to move `thirdparty/gtest` to `thirdparty/gtest/gtest` and move all this code which sets gtest settings to `thirdparty/gtest/CMakeLists.txt`. In this case you don't have to duplicate all this stuff

Reply:
Sounds great! Should it be done for other 3rdparties? (gflags && pugixml)

Reply:
pugixml does not need any configuration I suppose.

But for gflags we can do the same

Reply:
Done